token/js/src/extensions/interestBearingMint/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './instructions';
+export * from './state';
```

Comment:
Could you also add an `actions.ts` file with an `updateRate` implementation? You can see examples of that in the other extensions.